buildSrc/src/main/kotlin/androidx/build/ftl/GCloudCLIWrapper.kt

```diff
@@ -44,53 +46,103 @@ internal class GCloudCLIWrapper(
     /**
      * Path to the gcloud executable, derived from `which gcloud` call.
      */
-    private val executable: String by lazy {
+    private val gcloud: String by lazy {
+        findExecutable("gcloud")
```

Comment:
If gcloud is already on PATH, why do we need to find its exact location explicitly? Does execOperations not do that for us?

Reply:
actually i didn't try that. I can try it but looking for the executable lets me print a better error. Otherwise, we would need to parse the output of any call for "not-found" executable. If you don't mind, i'd rather keep it this way.